yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnSparkHadoopUtil.scala

```diff
@@ -519,9 +519,9 @@ object YarnSparkHadoopUtil {
       conf: SparkConf,
       numExecutors: Int = DEFAULT_NUMBER_EXECUTORS): Int = {
     if (Utils.isDynamicAllocationEnabled(conf)) {
-      val minNumExecutors = conf.get(DYN_ALLOCATION_MIN_EXECUTORS)
-      val initialNumExecutors = conf.get(DYN_ALLOCATION_INITIAL_EXECUTORS)
-      val maxNumExecutors = conf.get(DYN_ALLOCATION_MAX_EXECUTORS)
+      val minNumExecutors = Utils.getDynamicAllocationMinExecutors(conf)
```

Comment:
Hmm, no. `sparkConf.get(DYN_ALLOCATION_MAX_EXECUTORS)` will return you an int, and if it's not defined in the conf, it will return you the default value, so you won't need to duplicate it in your code.